md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ext/HiveTableMigrationAPI.java

```diff
@@ -0,0 +1,369 @@
+            com.wipro.ats.bdre.md.dao.jpa.Process sourcestageloadProcess = new com.wipro.ats.bdre.md.dao.jpa.Process();
+            com.wipro.ats.bdre.md.dao.jpa.Process sourcetodeststagecopyProcess = new com.wipro.ats.bdre.md.dao.jpa.Process();
+            com.wipro.ats.bdre.md.dao.jpa.Process desttableloadProcess = new com.wipro.ats.bdre.md.dao.jpa.Process();
+            com.wipro.ats.bdre.md.dao.jpa.Process registerpartitionProcess = new com.wipro.ats.bdre.md.dao.jpa.Process();
+
+
+            preprocessingProcess = Dao2TableUtil.buildJPAProcess(32, "PreProcessing of table-" + i, "preprocessing:table-" + i, 1, busDomainID);
+            sourcestageloadProcess = Dao2TableUtil.buildJPAProcess(33, "source stage load of table-" + i, "sourcestageload:table-" + i, 1, busDomainID);
+            sourcetodeststagecopyProcess = Dao2TableUtil.buildJPAProcess(34, "src-dest stagecopy of table-" + i, "sourcetodeststagecopy:table-" + i, 1, busDomainID);
+            desttableloadProcess = Dao2TableUtil.buildJPAProcess(35, "dest table load of table-" + i, "desttableload:table-" + i, 1, busDomainID);
+            registerpartitionProcess = Dao2TableUtil.buildJPAProcess(36, "register partition of table-" + i, "registerpartition:table-" + i, 1, busDomainID);
+            childProcesses.add(preprocessingProcess);
+            childProcesses.add(sourcestageloadProcess);
+            childProcesses.add(sourcetodeststagecopyProcess);
+            childProcesses.add(desttableloadProcess);
+            childProcesses.add(registerpartitionProcess);
+
+            LOGGER.info("childprocess size" + childProcesses.size());
+            LOGGER.info("Properties size" + propertiesList.size());
+            List<com.wipro.ats.bdre.md.dao.jpa.Process> processList = processDAO.createHiveMigrationJob(parentProcess, childProcesses, propertiesList);
+            LOGGER.info("after method size" + processList.size());
+            List<com.wipro.ats.bdre.md.beans.table.Process> tableProcessList = new ArrayList<com.wipro.ats.bdre.md.beans.table.Process>();
+            tableProcessList = Dao2TableUtil.jpaList2TableProcessList(processList);
+            Integer counter = tableProcessList.size();
+            LOGGER.info(counter + "counter");
+            for (com.wipro.ats.bdre.md.beans.table.Process process : tableProcessList) {
+                process.setCounter(counter);
+                process.setTableAddTS(DateConverter.dateToString(process.getAddTS()));
+                process.setTableEditTS(DateConverter.dateToString(process.getEditTS()));
+                allTableProcessList.add(process);
+            }
+
+        }
+        restWrapper = new RestWrapper(allTableProcessList, RestWrapper.OK);
+        LOGGER.info("Process and Properties for data load process inserted by" + principal.getName());
+
+        return restWrapper;
+
+    }
+
+  /*  public static Map<Integer, String> getAllColumnsMap(String dbName, String tableName) throws ETLException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)